mobile/lib/state/audio_handler_cubit.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:audio_service/audio_service.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../components/radio_player/audio_player_handler.dart';
+
+class AudioHandlerCubit extends Cubit<AudioHandler?> {
+  /*
+  initializing AudioHandlerCubit with null because proper init is asynchronous,
+  but cubit init has to be synchronous, so it needs something to store
+   */
+  AudioHandlerCubit() : super(null) {
+    initAudioHandler(); // proper 'init function' call
+  }
+
+  Future<void> initAudioHandler() async {
+    final audioService = await AudioService.init(
+      builder: AudioPlayerHandler.new,
+      config: const AudioServiceConfig(
+        androidNotificationChannelName: 'Live radio stream',
+        androidNotificationOngoing: true,
+      ),
+    );
+    // setting proper AudioService object as state, notifying listeners
+    emit(audioService);
+  }
+
+  AudioHandler? get() {
+    return state;
+  }
```

Comment:
Unnecessary - there already is a public getter, can delete (if it's not intended to be further modified or overriden - if so, needs a clarifying comment)
```suggestion
```